www/js/incident/post-trip-prompt.js

```diff
@@ -159,27 +159,18 @@ angular.module('emission.incident.posttrip.prompt', ['emission.plugin.logger'])
     });
     $window.cordova.plugins.notification.local.on('trigger', function (notification, state, data) {
         // alert("triggered, no action");
-        Logger.log("triggered, no action");
-        if (!$ionicPlatform.is('ios')) {
-            Logger.log("notification is displayed even when app is in foreground, ignoring trigger");
-            return;
-        }
+        Logger.log("Notification triggered");
         if (!checkCategory(notification)) {
             Logger.log("notification "+notification+" is not an mode choice, returning...");
             return;
         }
         cleanDataIfNecessary(notification, state, data);
-        promptReport(notification, state, data).then(function(res) {
```

Comment:
Should you keep the `promptReport` for iOS? Otherwise if the app is in the foreground at the time that a trip ends, the user will magically go to the survey screen without pressing anything and without any way to stop it. Is that really OK behavior?